src/test/scala/UnitSpec.scala

```diff
@@ -0,0 +1,7 @@
+
+import org.scalatest._
+import org.scalatest.FlatSpec
+
+class UnitSpec extends FlatSpec with  Matchers with OptionValues with Inside with Inspectors {
```

Comment:
it should be an abstract class, and also you don't really need the {} if there's nothing in the class body